### DISCUSSION TITLE
Not able to install chain link package

### DISCUSSION THREAD

**Discussion by hoBabu1** [Cyfrin/foundry-full-course-cu#1455] [Q&A]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/128292819/7530cc29-3bb7-4f97-b319-2ca8a143566d)
please help me to get out of it .

**Comment by EngrPips** [comment]
Hello @hoBabu1 , This error would always occur whenever your internet connection is not stable, please ensure you network connection is stable and then try again.
